presto-main/src/main/java/io/prestosql/operator/TableScanOperator.java

```diff
@@ -299,6 +302,7 @@ public Page getOutput()
         }
 
         Page page = source.getNextPage();
+
```

Comment:
revert, especially that there are no other changes in the file

Reply:
Reverted.